client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal, FormGroup, ControlLabel, FormControl, Alert } from 'react-bootstrap';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import moment from 'moment';
+import axios from 'axios';
+import _ from 'lodash';
+
+const { func, bool, array } = PropTypes;
```

Comment:
Does it make sense to pollute global name space with `func`, `bool` and `array` only to use them once?
Especially that `PropTypes.array` reads better.